apps/condo/domains/organization/components/OrganizationSelect.tsx

```diff
@@ -0,0 +1,27 @@
+import { Select } from 'antd'
+import { OrganizationEmployee } from '@condo/domains/organization/utils/clientSchema'
+import React from 'react'
+import { useOrganization } from '@core/next/organization'
+
+export const OrganizationSelect = () => {
+    const { link, selectLink } = useOrganization()
+    const { objs: userOrganizations } = OrganizationEmployee.useObjects({})
+
+    const options = userOrganizations.map((organization) => {
+        const { value, label } = OrganizationEmployee.convertGQLItemToFormSelectState(organization)
+
+        return (<Select.Option key={value} value={value} title={label}>{label}</Select.Option>)
+    })
+
+    const handleChange = (value) => {
+        selectLink({ id: value })
+    }
+
+    return (
+        link && (
+            <Select value={link.id} size={'middle'} onChange={handleChange} style={{ width: '100%' }}>
+                {options}
+            </Select>
+        )
+    )
```

Comment:
As for me, I would not mix presentation and data transformation logic.

Take a look at my variant:

```jsx
const options = userOrganizations.map(OrganizationEmployee.convertGQLItemToFormSelectState)

const handleChange = (value) => {
    selectLink({ id: value })
}

return (
    link && (
        <Select value={link.id} size={'middle'} onChange={handleChange} style={{ width: '100%' }}>
            {options.map(({label, value}) => (
                <Select.Option key={value} value={value} title={label}>{label}</Select.Option>
            ))}
        </Select>
    )
)
```